draft-ietf-masque-connect-ip.md

```diff
@@ -112,6 +112,8 @@ HTTP Extended CONNECT as described in {{!EXT-CONNECT2=RFC8441}} and
 {{!EXT-CONNECT3=RFC9220}}. When using HTTP/1.x {{H1}}, it uses HTTP Upgrade
 as defined in {{Section 7.8 of HTTP}}.
 
+This document updates {{CONNECT-UDP}}.
```

Comment:
What's the purpose of this PR? IETF requires updating another document to be mentioned in the Abstract, but there is no such requirement in the Introduction. The requirement is silly and makes documents worse, but we abide by it because that is the law. Let's not make the situation worse by also polluting the Introduction.

Reply:
To my knowledge (and per the shepherd question template) it's required to mention it into the abstract _and_ introduction, but I can't find the citation to prove it.

Reply:
Yes, and an one reference is this: https://authors.ietf.org/en/required-content scroll down to the Introduction part that says:

 If the I-D intends to obsolete or update a previous RFC, ensure the Introduction briefly explains what is being updated and why.

Reply:
Thanks for finding the documentation @gloinul. If the rule are so, we shall abide.

Reply:
Sorry @DavidSchinazi -- I agree that it seems to not add much value, but thems the rules! 🤷‍♂️